bin/init.gradle

```diff
@@ -24,6 +24,17 @@ allprojects { everyProj ->
 
                         cp.add(v.javaCompile.destinationDir.getAbsolutePath())
                     }
+
+                    // apply config-attributes in classpath recovery
+                    if (project.hasProperty('confAttrs')) {
+                        def confAttrSpec = (project.hasProperty('confAttrs') 
```

Comment:
Why do we need to check again for `project.hasProperty('confAttrs')`? 
I think you can remove the check here and stay with `confAttrs.toLowerCase().split(',').collectEntries { [(it.split(':')[0])`

Reply:
I fixed that.